docs/src/design/stack/op_constraints.md

```diff
@@ -195,8 +195,8 @@ This group contains operations which require constraints with degree up to $5$.
 | ------------ | :----------: | :-------------: | :-------------------------------------:| :---------: |
 | `MRUPDATE`   | $96$         | `11_00000`      | [Crypto ops](./crypto_ops.md)          | $4$         |
 | `PUSH`       | $100$        | `11_00100`      | [I/O ops](./io_ops.md)                 | $4$         |
-| `<unused>`   | $104$        | `11_01000`      |                                        | $4$         |
-| `<unused>`   | $108$        | `11_01100`      |                                        | $4$         |
+| `SYSCALL`    | $104$        | `11_01000`      | [Flow control ops](../decoder/main.md) | $4$         |
+| `CALL`       | $108$        | `11_01100`      | [Flow control ops](../decoder/main.md) | $4$         |
```

Comment:
We also need to update the definition of $f_{ctrl}$ flag on line 258 to include `CALL` and `SYSCALL` flags.

Reply:
done